src/commands/createWebApp/readAzConfig.ts

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import * as os from 'os';
+import * as path from 'path';
+
+export async function readAzConfig(...properties: AzConfigProperties[]): Promise<AzConfig> {
```

Comment:
A comment clarifying what this is doing and where it is reading the config from would be helpful, I think.